shell/platform/darwin/common/framework/Headers/FlutterChannels.h

```diff
@@ -143,9 +143,46 @@ FLUTTER_DARWIN_EXPORT
  * Adjusts the number of messages that will get buffered when sending messages to
  * channels that aren't fully set up yet.  For example, the engine isn't running
  * yet or the channel's message handler isn't set up on the Dart side yet.
+ *
+ * @param name The channel name.
+ * @param messenger The binary messenger.
+ * @param newSize The new size of the buffer.
```

Comment:
"size" needs a clear description here. Bytes? kb? Messages?

Even better would be a parameter name that actually captures that information as well.